python/kfserving/kfserving/api/kf_serving_watch.py

```diff
@@ -48,15 +55,25 @@ def watch(name=None, namespace=None, timeout_seconds=600):
         else:
             if isvc.get('status', ''):
                 url = isvc['status'].get('url', '')
-                default_traffic = isvc['status'].get('traffic', '')
-                canary_traffic = isvc['status'].get('canaryTraffic', '')
+                if version == 'v1alpha2':
```

Comment:
we won't be able to show v1alpha2 status anymore, it is converted to v1beta1 and only shows v1beta1 status type. This code can work with 0.4 but in SDK we do not know which KFS version they are running, so probably not worth supporting v1alpha2 case, we can say SDK 0.5 only works with 0.5 KFS.

Reply:
OK, so Let's only support v1beta1 for the watching API.